utils/collect-updates/lib/get-packages-for-option.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+module.exports = getPackagesForOption;
+
+function getPackagesForOption(option) {
```

Comment:
Changed `getForcedPackages` to this more generic function for use with options besided `--force-publish`.

Reply:
Good call